iocore/net/Makefile.am

```diff
@@ -53,6 +53,7 @@ test_certlookup_SOURCES = \
 test_certlookup_LDADD = \
 	$(top_builddir)/iocore/eventsystem/libinkevent.a \
 	$(top_builddir)/src/tscore/libtscore.a \
+	$(top_builddir)/src/api/libtsapicore.a \
```

Comment:
I'm pretty sure tsapicore.a need to become dynamically loaded and installed.  @dragon512 can confirm.